tools/mo/openvino/tools/mo/utils/versions_checker.py

```diff
@@ -287,9 +285,17 @@ def check_requirements(framework=None):
             continue
 
     if len(not_satisfied_versions) != 0:
-        extension = 'bat' if os.name == 'nt' else 'sh'
-        install_file = 'install_prerequisites{0}.{1}'.format(framework_suffix, extension)
-        helper_command = os.path.join(os.path.dirname(requirements_file), 'install_prerequisites', install_file)
+
+        if framework == 'tf':
```

Comment:
Why Caffe and other supported frameworks are not covered?
https://github.com/openvinotoolkit/openvino/blob/bc70b2b68b849503815466a30a14333a2b32b12b/tools/mo/openvino/tools/mo/main.py#L236